github-backup.py

```diff
@@ -8,15 +8,30 @@
 """
 
 
-from github import Github
-from argparse import ArgumentParser
+import os
+import errno
+import codecs
+import json
+import itertools
 import subprocess
-import os, os.path
 import logging
+import getpass
+try: #PY3
+    from configparser import SafeConfigParser as ConfigParser
+except ImportError:
+    from ConfigParser import SafeConfigParser as ConfigParser
+from argparse import ArgumentParser
+
+import requests
+import github
 
 LOGGER = logging.getLogger('github-backup')
 
+IsAuthorized = False
```

Comment:
Please check code style. We used lowercase naming with underscores in the code so far.

Reply:
I did a different coding style because its a global variable. I'll change it to allow upper case like the global LOGGER.